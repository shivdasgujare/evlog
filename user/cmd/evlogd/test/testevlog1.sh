#!/bin/bash
#
ARCH=i386
TEST_TEMPLATE=testevlog1.out

/sbin/evlconfig -L 3 -d on -i 10 -c 100
/sbin/evlsend -f LOCAL1 -t 100 -m "some junk1"
/sbin/evlsend -f LOCAL1 -t 100 -m "some junk2"
sleep 1
logtime=`date +%s`
sleep 1
./testdup3 3 3 1 0
sleep 1
/sbin/evlsend -f LOCAL1 -t 100 -m "some junk3"
sleep 5
/sbin/evlview -f "(time>${logtime} && event_type=0x7d1)" -B -S 'size=%size%\n%data%' > testevlog1.sh.out
/sbin/evlview -f "(time>${logtime} && event_type=0x7d2)" -B -S 'size=%size%\n%data%' >> testevlog1.sh.out
/sbin/evlview -f "(time>${logtime} && event_type=0x7d3)" -B -S 'size=%size%\n%data%' >> testevlog1.sh.out
/sbin/evlview -f "(time>${logtime} && facility=LOGMGMT && event_type=1)" -B -S 'size=%size%\n%data%' >> testevlog1.sh.out

diff testevlog1.sh.out $TEST_TEMPLATE

if [ $? -eq "0" ]
then
	echo testevlog1		:PASSED
else
	echo testevlog1		:FAILED
fi
