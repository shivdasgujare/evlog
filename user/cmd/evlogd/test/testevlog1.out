size=18
Test duplicate #1
size=18
Test duplicate #2
size=18
Test duplicate #3
size=68
Discarded 2 duplicate events, event_type = 0x7d1, facility = LOCAL0
size=68
Discarded 2 duplicate events, event_type = 0x7d2, facility = LOCAL0
size=68
Discarded 2 duplicate events, event_type = 0x7d3, facility = LOCAL0
